platform-dependent-components/problem-solver/cxx/generate-response-module/agent/GenerateResponseAgent.cpp

```diff
@@ -69,8 +75,9 @@ void GenerateResponseAgent::attachAnswer(
     ScAddr const & messageAddr,
     ScAddr const & answerAddr)
 {
-  utils::GenerationUtils::generateRelationBetween(&m_memoryCtx, messageAddr, messageAnswer, Keynodes::nrel_reply_structure);
-  m_memoryCtx.CreateEdge(ScType::EdgeAccessConstPosTemp, answerAddr, messageAnswer);
+  utils::GenerationUtils::generateRelationBetween(
+      &m_memoryCtx, messageAddr, messageAnswer, Keynodes::nrel_reply_structure);
```

Comment:
fix documentation please, this keynode is not in structure
![image](https://github.com/user-attachments/assets/3314281d-91b0-4c9d-b29a-a73e2b589789)